__tests__/app.test.js

```diff
@@ -228,4 +228,51 @@ describe('/api/articles/:article_id/comments', () => {
             expect(error).toEqual({ err: 400, msg: 'Bad request'});
         })
     });
+});
+
+describe('/api/articles/:article_id', () => {
+    test('PATCH: 200 updates article votes when given an object indicating how much to either increase or descrease the votes by', () => {
+        const id = 1;
+        const newVotes = { inc_votes: -5 }
+        return request(app)
+        .patch(`/api/articles/${id}`)
+        .send(newVotes)
+        .expect(200)
+        .then((response) => {
+            const updatedArticle = response.body;
+            expect(updatedArticle.votes).toEqual(95);
```

Comment:
This would be a more robust test if you used `.toMatchObject` and expected the whole article back with the correct article_id and votes values